app/src/web.ts

```diff
@@ -0,0 +1,40 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type { AppInfo, AppPlugin, AppLaunchUrl, AppState } from './definitions';
+
+export class AppWeb extends WebPlugin implements AppPlugin {
+  constructor() {
+    super({ name: 'App' });
+    if (typeof document !== 'undefined') {
+      document.addEventListener(
+        'visibilitychange',
+        this.handleVisibilityChange.bind(this),
+        false,
+      );
+    }
+  }
+
+  exitApp(): never {
+    throw new Error('Method not implemented.');
+  }
+
+  async getInfo(): Promise<AppInfo> {
+    throw new Error('Method not implemented.');
```

Comment:
```suggestion
    throw this.unimplemented('Not implemented on web.');
```

Reply:
Hmm... it seems to me it would be more useful (and developer-friendly) to at least attempt to return something useful on the web. I see no technical reason why it can't at least the name and version from package.json.

Reply:
@dwieeb Any thoughts?

Reply:
We've talked about doing that in the future. We don't have access to `package.json` in the web runtime, but the Capacitor CLI could write that info somewhere where it's available.